Creiwry/kitten_project
# feat(backend/models): add after_create create_cart method to user model

### Ticket Link
https://trello.com/c/DBPyHMEU

### User Story
N/A

### What does this PR do?
It adds a create_cart method to the user model and calls it after the creation of each user

### Where should the reviewer start?
The reviewer should start in the user model file. A test is also written for this functionality
Reviewer should run
```bash
rspec spec/models/user_spec.rb
```
to see if the test passes

### How should this be manually tested?
In rails console, the reviewer can run

```ruby
user = User.create(email: "ray.toro@mychem.com", password: "secret")
```

Then, the reviewer should run
```ruby
user.cart
```
This should return an active records entry

### Are there any breaking changes?
No
